### DISCUSSION TITLE
'""C:\Program' is not recognized as an internal or external command error when installed dependencies are ran on windows

### DISCUSSION THREAD

**Discussion by MickyG03** [dipy/dipy#2784] [3. Community Lounge]
## Description
After installing dipy dependencies in windows, I faced some issues like, if I run "dipy_horizon", the cmd window shows:

'""C:\Program' is not recognized as an internal or external command, operable program or batch file.

I believe there is some issue with inverted comas in the batch files.

## Way to reproduce
I used this link for installation of dependencies in windows os:
https://dipy.org/documentation/1.6.0./installation/

-after installing using this - "python setup.py install" or "python setup.py develop" in cmd and then running dipy_horizon or anyother dependency, we get this error

![image](https://user-images.githubusercontent.com/76037226/232961923-ccfcc95e-847d-4b63-a984-32b6c84afd90.png)

(Also installation was successfull, and all build tools were installed)

- Operating system and version: Windows-10-10.0.19044-SP0
- Python version: Python 3.10.6
- dipy version:1.7.0dev0

**Comment by pjsjongsung** [comment]
Are you still having this issue? This is very strange.
If you are, can you try updating dipy and fury using `pip install -U dipy fury` and try again?
We had a recent release on both.

**Reply by Garyfallidis** [reply]
And uninstall previous versions?

**Comment by Garyfallidis** [comment]
Ah wait. Use anaconda prompt and it seems your system does not know the path for dipy_horizon in windows.
